src/modules/account/RadixAccountSystem.ts

```diff
@@ -3,4 +3,5 @@ import { RadixAtomObservation } from '../..'
 export default interface RadixAccountSystem {
     name: string
     processAtomUpdate(atomObservation: RadixAtomObservation)
+    getState()
```

Comment:
No type constraints at all? Then it should be explicitly any at least

Reply:
I can't put a type constraint in the interface yet, since not all of the systems have been implemented. I do however have a type in the token definition and transfer systems, that has been refactored so far.